apps/app/lib/app/auth/webhook.ex

```diff
@@ -0,0 +1,24 @@
+defmodule App.Auth.Webhook do
+  alias Plug
+
+  def init(opts) do
+    opts
+  end
+
+  def call(conn, opts) do
+    hash =
+      Plug.Conn.get_req_header(conn, "x-shopify-hmac-sha256")
+      |> List.first()
+
+    secret_key = "153a97df2d7168c3c05b62c49ff83595763d9d11cff7e4bc3c37f88f8092698a"
```

Comment:
This has to become an environment variable. Example: System.get_env("SOME_ENV"). When you make this change tell me to add it to nanobox.